index.html

```diff
@@ -1412,6 +1412,153 @@ <h3>Biometrics</h3>
+        section of the Ethical Web Principles, there are certain key goals that should 
+        apply to all Web standards and technologies. DIDs explicitly support several
+        of these goals, especially the following:
+        <ul>
+          <li><a href="https://www.w3.org/2001/tag/doc/ethical-web-principles/#privacy">Privacy</a></li>
+          <li><a href="https://www.w3.org/2001/tag/doc/ethical-web-principles/#control">Individual Control</a></li>
+          <li><a href="https://www.w3.org/2001/tag/doc/ethical-web-principles/#multi">Device Independence</a></li>
+        </ul>
+      </p>
+    </section>
+
+    <section class="informative">
+      <h3>Energy Usage and Environmental Impacts</h3>
+      <p class="advisement">
+        The following section reflects the views of some members of the working group.
+        Additional PRs are welcome from the working group with additional points of view.
+      </p>
+
+      <section class="informative">
+        <h3>Environmental Principles as Applied to DID method development</h3>
+        <p>
+          When implementing or utilizing a DID method, consideration should be given
+          to the environmental impacts of any underlying technologies.  
+          Avoidance of excess energy use that does not further other equally important 
+          goals such as human rights or personal privacy, particularly when that energy 
+          is sourced from non-sustainable energy creation methods, is a best 
+          practice that serves the interest of all.
+        </p>
+  
+        <p>
+          Direct comparison of each factor, and the balances between them, should be
+          considered when evaluating particular methods or approaches. Notably, the
+          ability to misuse or control DIDs that could impede privacy might be
+          found to be inversely correlated with additional energy consumption.
+        </p>
+        
+        <p class="advisement">
+          The guiding principle that 
+          <a href="https://www.w3.org/2001/tag/doc/ethical-web-principles/#sustainable">the
+          web must be an environmentally sustainable platform</a> should be followed.
+          <br />As part of following this principle,    
+          utilizing or authoring DID methods that require unsustainably-sourced 
+          energy as part of their technical implementation or utilization should
+          be very carefully considered in balance against alternative approaches.
+        </p>
+  
+        <p>
+          When selecting a DID method, the method that minimizes energy consumption 
+          while meeting privacy, interoperability, and other technical requirements
+          should generally be chosen.
+        </p>
+      </section>
+
+      <section class="informative">
+        <h3>Environmental Impact Assesment</h3>
+        
+        <p>
+          The energy usage and other environmental impacts of a DID-based solution
```

Comment:
Strike "energy use" here. It's not appropriate to call that out. ALL environmental impacts should be considered equally, without elevating any. It is that selective elevation that makes these comments political and inappropriate.